src/handlers/metadataDelta/handlers/addVerifiedMembers.ts

```diff
@@ -43,6 +46,25 @@ export const handleAddVerifiedMembers = async (
         id: getColonyContributorId(colonyAddress, userAddress),
       });
 
+      // If a wallet address is not currently a member of a Colony,
+      // add the entity as a new Colony contributor
+      if (item?.data?.getColonyContributor === null) {
```

Comment:
I think we could do this with one less GQL call, by checking if it's null, then creating a contributor with `isWatching` set to `true` otherwise do the usual flow of updating the contributor document

Reply:
> I think we could do this with one less GQL call, by checking if it's null, then creating a contributor with isWatching set to true 

I tried to do that yesterday but the block-ingestor was throwing all sorts of errors I'm not exactly sure why 😕 I deleted the containers, ran my dev and even restarted my laptop! But the same issue kept coming up. Then I did this instead and it was working so I stuck with it.

Reply:
See this commit @bassgeta https://github.com/JoinColony/block-ingestor/pull/230/commits/7a056883458a945d9245fbd285ce8206f627020a

Reply:
@bassgeta I'll make sure to run the block-ingestor right after amplify is finished! I've reverted back to the changes I made to set `isWatching` to `true` on Contributor creation!

Reply:
@bassgeta I have also updated the behaviour for new Contributors who were added via the Manage Verified Members flow. I added a screenshot of the conversation I had with Arren about this. Apologies, can you review again? Thank you! 🙏